Examples/Algorithms/TrackFitting/src/GsfFitterFunction.cpp

```diff
@@ -106,23 +111,38 @@ std::shared_ptr<TrackFittingAlgorithm::TrackFitterFunction>
 ActsExamples::makeGsfFitterFunction(
     std::shared_ptr<const Acts::TrackingGeometry> trackingGeometry,
     std::shared_ptr<const Acts::MagneticFieldProvider> magneticField,
+    std::string lowBetheHeitlerPath, std::string highBetheHeitlerPath,
```

Comment:
should we box these string in an optional to be more explicit?

Reply:
ah yeah :D see the comment above